titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/volume/SharedContainerVolume.java

```diff
@@ -0,0 +1,51 @@
+package com.netflix.titus.api.jobmanager.model.job.volume;
+
+import com.netflix.titus.api.jobmanager.model.job.SharedContainerVolumeSource;
+
+public class SharedContainerVolume extends Volume {
+
+    private final SharedContainerVolumeSource sharedContainerVolumeSource;
```

Comment:
Isn't `SharedContainerVolumeSource` and `SharedContainerVolume` the same? Why not just `SharedContainerVolume` with fields from `SharedContainerVolumeSource`?

Reply:
Yes, you are right. I didn't really understand "What" we were inheriting till now.
Next pass will just be the SharedContainerVolumeSource with the right fields.